src/script_fmt/genus_tcl.tera

```diff
@@ -1,8 +1,8 @@
 # {{ HEADER_AUTOGEN }}
-if [ info exists search_path ] {{ '{{' }}
+if [ info exists search_path ] {{ '{' }}
   set search_path_initial $search_path
-{{ '}}' }}
-set ROOT = "{{ root }}"
+{{ '}' }}
```

Comment:
I am unsure if the change from two braces to one is needed, but I do not use genus, so I trust you tested this.

Another difference to the previous script prior to the use of templates, the `search_path_initial` variable used to be declared in an else statement as well, it might make sense to add this back: https://github.com/pulp-platform/bender/blob/25c518464b6a0c098e815c84f055e8fc2a4829f1/src/cmd/script.rs#L847-L852

Reply:
The script fails with two braces, Genus then interprets it as a string.

The else branch for `search_path_initial` does not really matter. I have added it anyway.

Reply:
Ok, thanks!